### PR TITLE
add support for coconut (functional python) kernel

### PR DESCRIPTION
**Pull request by kksgandhi**

### CODE REVIEW

**Comment by kksgandhi** [comment]
This references issue #100

**Comment by kksgandhi** [comment]
It seems like the REPL for coconut is exactly the same as python, so I was able to copy over the python language class to serve as the coconut language class in `python3/language.py`

**Comment by kksgandhi** [comment]
For the coconut filetype plugin, I just copied the python filetype plugin. If there's a more intelligent way to handle this (e.g. something like `source ../python/jupyter.vim`) then I'm all ears.

**Comment by wmvanvliet** [comment]
You can do `runtime! <sfile>:h/../julia/*.vim`. This command will source all the vim files in a path relative to the current file.

**Comment by wmvanvliet** [comment]
See also https://vi.stackexchange.com/questions/3177/use-single-ftplugin-for-more-than-one-filetype

**Comment by kksgandhi** [comment]
Thanks @wmvanvliet, made those changes!

File: ftplugin/coconut/jupyter.vim [hunk 1]
```diff
@@ -1,60 +1,16 @@
...
-    nnoremap <buffer> <silent> <localleader>I :PythonImportThisFile<CR>
-
-    " Debugging maps
-    nnoremap <buffer> <silent> <localleader>b :PythonSetBreak<CR>
-endif
-"}}}
-
-"=============================================================================
-"=============================================================================
+runtime! ftplugin/python/jupyter.vim
```
Comment (wmvanvliet) [inline ftplugin/coconut/jupyter.vim:16]: As I understand it, `runtime` searches all the folders in the runtime path. So if a user has their own `.vim/ftplugin/python/jupyter.vim` file somewhere, that will get sourced too.

**Review by wmvanvliet — CHANGES_REQUESTED** [review]

File: ftplugin/coconut/jupyter.vim (hunk 1 above)
Comment (wmvanvliet) [reply ftplugin/coconut/jupyter.vim:16]: So I do think we want to use a relative path here:

```suggestion
runtime! <sfile>:h/../python/jupyter.vim
```

**Comment by wmvanvliet** [comment]
Works great. Thanks a lot @kksgandhi!

**Review by wmvanvliet — APPROVED** [review]

**Comment by kksgandhi** [comment]
Thank you so much!

**Pull request merged**